docs/pages/config-reference.mdx

```diff
@@ -156,6 +156,12 @@ auth_service:
     # certificates and keys (may need to wipe out /var/lib/teleport directory)
     cluster_name: "main"
 
+    # ProxyProtocol enables support for HAProxy proxy protocol when it is turned 'on'.
```

Comment:
```suggestion
    # ProxyProtocol enables support for HAProxy proxy protocol version 1 when it is turned 'on'.
```

Teleport doesn't support v2 (the binary version), only v1 (the plain text version). Would be useful for people to know.